prime-router/src/test/kotlin/common/UniversalPipelineTestUtils.kt

```diff
@@ -257,12 +257,13 @@ object UniversalPipelineTestUtils {
     )
 
     /**
-     * fetch child reports associated with a [parent] report and ensure we find an [expected] number of children
+     * fetch child reports associated with a [parent] report and ensure we find an [expectedItems] number of children
      */
     fun fetchChildReports(
         parent: Report,
         txn: DataAccessTransaction,
-        expected: Int? = null,
+        expectedItems: Int? = null,
+        expectedReports: Int = 1,
```

Comment:
This needed to get adjusted since its now possible to generate a report with no lineages (i.e. an empty report).